addons/OPT/GPS/fn_clientInitEH.sqf

```diff
@@ -24,6 +24,7 @@
 
 #include "macros.hpp";
 
+/*
```

Comment:
Am Besten löschen statt auskommentieren. Git merkt sich ja, was da war, sodass man das ganz einfach wieder her zaubern kann, wenn man es wieder braucht.

Reply:
kann man machen wenn man es irgendwo wieder her bekommt.

Reply:
In Git geht nie etwas verloren, da sämtliche Änderungen protokolliert werden. Du kannst also ohne Bedenken löschen ;)